1.0-E2E-status.md

```diff
@@ -115,31 +115,33 @@ Legend:
-|                              | C#           | Dapeng    | :hourglass_flowing_sand: | -       | -                                                                                            |                                                               |
-|                              | JS           | Qiaoqiao  | :hourglass_flowing_sand: | -       | -                                                                                            |                                                               |
-|                              | Java         | Weidong   | :hourglass_flowing_sand: | -       | -                                                                                            |                                                               |
-|                              | Python       | Yuchao    | :hourglass_flowing_sand: | -       | -                                                                                            |                                                               |
+| **Specs**                    | **Language** | **Owner** | **Status**               | **ETA** | **Link** | **Remarks** |
+| ---------------------------- | ------------ | --------- | ------------------------ | ------- | -------- | ----------- |
+| Overall Language Feature Gap | -            | -         |                          | -       | -        |             |
+| PetStore                     |              |           |                          |         |          |             |
+|                              | C#           | Dapeng    | :green_circle:           | 12/31   | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/petstore/clients/dotnet) |             |
+|                              | JS           | Qiaoqiao  | :running_man:            | 12/31   | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/petstore/clients/javascript)       | needs to upgrade `ts-http-runtime` to preview version            |
+|                              | Java         | Weidong   | :running_man:            | 12/31   | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/petstore/clients/java) | `clientcore` not published to Maven |
+|                              | Python       | Yuchao    | :green_circle:           | 12/31   | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/petstore/clients/python) |             |
+| ToDo App                     |              |           |                          |         |          |             |
+|                              | C#           | Dapeng    | :running_man:            | 1/15    | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/todoApp/clients/dotnet) | `http-client-csharp` now has a bug on multi-line descriptions |
```

Comment:
Could you remove this?  'http-client-csharp` now has a bug on multi-line descriptions'   I didn't find that in the original file.

Reply:
updated